### DISCUSSION TITLE
Are AliasNames supported in Milo Namespace

### DISCUSSION THREAD

**Discussion by fatihalgan** [eclipse-milo/milo#1108] [General]
I was not able to see anything that corresponds to Aliases as explained in [https://reference.opcfoundation.org/Core/Part17/v105/docs/1](here). Does Milo have this feature, if so could someone point where to lok for it in the API?

**Comment by kevinherron** [comment]
No, Part 17 wasn't introduced until OPC UA 1.04 and the main branch of Milo is still OPC UA 1.03.

When 1.0 (`dev/1.0` branch) is released sometime this year it will support OPC UA 1.05 and all the new DataTypes, ObjectTypes, and ReferenceTypes introduced by Part 17 will exist.

**Reply by fatihalgan** [reply]
Thanks for the update Keivn.
